BuoyTest/src/components/cocktails/CocktailsList.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Stylesheet, View, FlatList, Text, Image, TouchableWithoutFeedback } from 'react-native';
+import styles from './styles';
+
+class CocktailsList extends Component {
+  state = {}
+  tableHeader = () => (
+    <View>
+    </View>
+  )
+
+  renderRows(object, indexRow) {
+    const cocktail = this.props.cocktails.find(cocktail => cocktail.idDrink === object.idDrink);
```

Comment:
The object parameter is already the cocktail that you need. This finding can be removed

Reply:
Updated!